app/models/enterprise.rb

```diff
@@ -199,7 +189,7 @@ def self.distinct_count
   end
 
   def activated?
-    confirmed_at.present? && sells != 'unspecified'
+    contact.confirmed? && sells != 'unspecified'
```

Comment:
Ok, let's move the `receives_notifications` discussion here then so that we have a thread to use. 

I am happy to change the spec, just wondering if @daniellemoorhead or @mkllnk have any objections? I had a chat about it with @sstead yesterday and she had no major objections but was not 100% sure it would work for one specific case. Can you point us to some documentation of this case @sstead?

For those who are catching up, this is the `receives_notifications` proposal:

>General comment: perhaps this is a later thing, but if we are going to use an association with a user to determine who will receive notification emails, perhaps we could just use the existing `managers` association instead by adding a `receives_notifications` flag to `enterprise_roles`? We could add a migration that turns the flag on for the user that matches the existing confirmed `email` field, and it could default off for any other managers, so that we preserve pretty much the existing state. That would add a bit of flexibility, and seems fairly logical to me.